<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Continuous-Time-Frequency Transform</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Transform</string>
     </property>
     <widget class="QLineEdit" name="l0line">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>300</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Window type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Window length:</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sampling frequency:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quickanddirtyButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quicky and dirty settings</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Step:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>STFT</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Wavelet</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_16">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Window type:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_8">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_17">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Order</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_18">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Discret scale</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>491</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Modenumber calculation</string>
     </property>
     <widget class="QLabel" name="firsttime_9">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Averaging</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-5</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_10">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Modenumber range</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="firsttime_11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Step:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l0line_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Should I do mode number calculation?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Toroidal</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Poloidal</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>292</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start calculation</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Plotting</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>541</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlSpectra"/>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Do plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="loadGB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Load signal</string>
    </property>
    <widget class="QPushButton" name="loadsignalButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load signal</string>
     </property>
    </widget>
    <widget class="QLabel" name="text1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of data point:</string>
     </property>
    </widget>
    <widget class="QLabel" name="text2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sampling frequency:</string>
     </property>
    </widget>
    <widget class="QLabel" name="datapointsLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="samplingfrequencyLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 kHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="text3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timerange:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timerangeLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 s</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="progressGB">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Progress-log</string>
    </property>
    <widget class="QTextEdit" name="progresslogTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="selectchannelsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select channels</string>
    </property>
   </widget>
   <widget class="QLabel" name="text4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of selected channel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="channelnumberLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
